kano_updater/scenarios.py

```diff
@@ -654,7 +663,7 @@ def _bootconfig_set_value_helper(self, setting, value):
     def beta_370_to_beta_380(self):
         # linux kernel 4.4.21 shipped with Kano 3.8.0 emits systemd boot messages.
         # fix by telling the kernel to enable an empty splash screen.
-        command="sed -i 's/\bsystemd.show_status=0\b/splash/' {}".format('/boot/cmdline.txt')
+        command = "sed -i 's/\bsystemd.show_status=0\b/splash/' {}".format('/boot/cmdline.txt')
```

Comment:
I changed this line yesterday to fix the bootup messages, on the `rc` branch.
Will we have troubles to bring it back here?

* https://github.com/KanoComputing/kano-updater/commit/d9fe7aff77553d314bd6f65cd7f4642e2f2f9d45

@radujipa @Ealdwulf